deep_speech_2/README.md

```diff
@@ -18,13 +18,17 @@
 - [Experiments and Benchmarks](#experiments-and-benchmarks)
 - [Questions and Help](#questions-and-help)
 
-## Prerequisites
-- Python 2.7 only supported
-- PaddlePaddle the latest version (please refer to the [Installation Guide](https://github.com/PaddlePaddle/Paddle#installation))
+
 
 ## Installation
 
-Please make sure the above [prerequisites](#prerequisites) have been satisfied before moving on.
+To avoid the trouble of environment setup, [running in docker container](#Running-in-Docker-Container) is highly recommended. Otherwise follow the guidelines below to install the dependencies manually.
```

Comment:
#Running-in-Docker-Container --> #running-in-docker-container
Otherwise, the hyperlink is not available.

Reply:
Modified